llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -774,13 +773,36 @@ define void @foo(ptr %arg0, ptr %arg1) {
   EXPECT_EQ(NewLd->getName(), "NewLd");
 
   sandboxir::LoadInst *NewVLd =
```

Comment:
nit: Let's make this consistent with the other tests by adding a comment like: `//  Check create(InsertBefore, IsVolatile=true`)

Reply:
fixed